src/plusMoins.java

```diff
@@ -1,7 +1,6 @@
 
```

Comment:
another back to old it seems. sorry but the upgread of the player managing system must be outside of the game function. The players managing system must be part of the option menu, as strange as it looks